<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWatersDialog</class>
 <widget class="QDialog" name="SelectionWatersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Year/Waters Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSelectYear">
       <property name="text">
        <string>Select Year</string>
       </property>
       <property name="buddy">
        <cstring>comboSelectYear</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboSelectYear"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSelectWaters">
       <property name="text">
        <string>Select Waters</string>
       </property>
       <property name="buddy">
        <cstring>listWaters</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listWaters">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
